serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/basic-integration-tests-with-restassured.adoc

```diff
@@ -1 +1,242 @@
+from the https://github.com/kiegroup/kogito-examples/tree/main/kogito-quarkus-examples/serverless-workflow-testing-with-rest-assured[GitHub repository].
+
+This guide describes how to test a Serverless Workflow application that serves a `hello` endpoint.
+The endpoint expects a `name` as a parameter and returns a `greeting`.
+
+Example: For a request with the given body:
+
+[source,json]
+----
+{
+  "workflowdata": {
+    "name": "John Doe"
+  }
+}
+----
+
+The endpoint should return a response with a body similar to:
+
+[source,json]
+----
+{
+  "id": "00ce5c1d-4936-4e18-998c-f5c5d5d784ad",
+  "workflowdata": {
+    "greeting": "Hello, John Doe"
+  }
+}
+----
+
+.Prerequisites
+* https://adoptium.net/[JDK 11+] is installed with `JAVA_HOME` configured appropriately.
+* https://maven.apache.org/install.html[Apache Maven] is installed.
+* https://quarkus.io/guides/cli-tooling[Quarkus CLI] is installed.
+* Visual Studio Code with https://marketplace.visualstudio.com/items?itemName=redhat.java[Red Hat Java Extension]
+and https://marketplace.visualstudio.com/items?itemName=redhat.vscode-extension-serverless-workflow-editor[Red Hat Serverless Workflow Editor] is installed to edit your workflows.
+
+For more information about the tooling and the required dependencies, see xref:getting-started/getting-familiar-with-our-tooling.adoc[tooling guide].
+
+[[boostrapping-the-project]]
+== Bootstrapping a project
+
+To create your Serverless Workflow service, first you need to bootstrap a project.
+
+.Procedure
+
+. In a command terminal, enter the following command to create a project:
++
+--
+.Create a project using Quarkus CLI
+[source,shell]
+----
+quarkus create app \
+    -x=kogito-quarkus-serverless-workflow \
+    -x=quarkus-resteasy-jackson \
+    org.kie.kogito.examples:serverless-workflow-testing-with-rest-assured:1.0
+----
+--
+
+For more information about bootstrapping a Serverless Workflow project, see
+xref:getting-started/create-your-first-workflow-service.adoc[Create Your First Serverless Workflow Service guide].
+
+[[creating-workflow]]
+== Creating a workflow
+
+After bootstrapping a project, you need to create a workflow. In the following procedure, a workflow named
+`Hello workflow` is created.
+
+.Procedure
+. Create a file named `hello.sw.json` in the `src/main/resources` directory with the following content:
++
+--
+.Example content for `hello.sw.json` file
+[source,json]
+----
+{
+  "id": "hello",
+  "version": "1.0",
+  "name": "Hello workflow",
+  "description": "JSON based hello workflow",
+  "start": "Hello",
+  "states": [
+    {
+      "name": "Hello",
+      "type": "inject",
+      "data": {
+        "hello": "Hello, "
+      },
+      "stateDataFilter": {
+        "output": "${ {greeting: .hello+.name} }"
+      },
+      "end": true
+    }
+  ]
+}
+----
+--
+
+For the workflow defined above, Kogito will create an endpoint at `/hello`, which is the `id` of the workflow.
```

Comment:
```suggestion
For the workflow defined above, the `/hello` will be created, which is the `id` of the workflow.
```

Reply:
@spolti, for reading purposes, it's better to avoid passive voice, IMO.

What about:

```suggestion
The engine creates an endpoint at the `/hello` path. The path naming is the same as the workflow `id` in the definition file.
```